### PR TITLE
Adjustments in logging and LOKI due to Observability cluster

### LINKED ISSUE #324: Setup Logging on the New Observability Cluster 2/2
- subtask of https://github.com/nerc-project/operations/issues/177
- Zenhub Epic [```Setup a new Cluster to handle Metrics, Logging and Observability```](https://app.zenhub.com/workspaces/moc-a--rh-sprint-workspace-62a210f69d42f600151deae0/epics/Z2lkOi8vcmFwdG9yL1plbmh1YkVwaWMvMTQzNzMy)

timeline: 2023/Q4 - 2024/Q1

right now catching this as an issue for #177

- [x] logging solution in #405
- [ ] log sources this issue

### PR DESCRIPTION
**Pull request by bnshr**
fixes https://github.com/nerc-project/operations/issues/324
Referring to [the issue](https://github.com/nerc-project/operations/issues/324)

### CODE REVIEW

File: logging/overlays/nerc-ocp-obs/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../base
+
+patchesStrategicMerge:
+  - externalsecrets/openshift-logging-lokistack-gateway-bearer-token_patch.yaml
```
Comment (larsks) [inline logging/overlays/nerc-ocp-obs/kustomization.yaml]: `patchesStrategicMerge` is deprecated syntax. This should read:

```suggestion
patches:
  - path: externalsecrets/openshift-logging-lokistack-gateway-bearer-token_patch.yaml
```

**Review by larsks — CHANGES_REQUESTED** [review]

File: logging/overlays/nerc-ocp-obs/kustomization.yaml (hunk 1 above)
Comment (bnshr) [reply logging/overlays/nerc-ocp-obs/kustomization.yaml]: Thank you! Is there a way to detect deprecated syntax by some tool/command?

Comment (larsks) [reply logging/overlays/nerc-ocp-obs/kustomization.yaml]: Just use a newer version of Kustomize:

```
$ kustomize version
v5.2.1
$ kustomize build logging/overlays/nerc-ocp-infra/ > /dev/null
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
```

Comment (bnshr) [reply logging/overlays/nerc-ocp-obs/kustomization.yaml]: Thanks. I changed all occurrences here.

**Comment by larsks** [comment]
@schwesig This PR is still marked and draft and also has conflicts that need to be resolved.

**Comment by schwesig** [comment]
> @schwesig This PR is still marked and draft and also has conflicts that need to be resolved.

my bad, I clicked the wrong button in zenhub :-(

**Pull request closed without merging**